salt/roster/sshknownhosts.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+'''
+Parses roster entries out of Host directives from SSH known_hosts
+
+Example of configuration,
+keep in mind that known_hosts file only contain hostname/IP,
+so you have to use roster_defaults for other parameter for each minion
```

Comment:
The previous two lines might read better like this:
```
.. note::

    The ``known_hosts`` file only contains hostname/IP. To pass other parameters, 
    use ``roster_defaults``.

```

Reply:
@remijouannet were you also going to address this change?

Reply:
Yeah I'm on my phone right now, so I'm gonna wait tomorrow when I have access to a computer

Reply:
Great! Looking forward to getting this merged! 🙂